sql/log_event.cc

```diff
@@ -1586,7 +1586,7 @@ Log_event* Log_event::read_log_event(const char* buf, uint event_len,
       if (!force_opt)
       {
         *error= "No point in reading encrypted binlog - quitting. "
-                "Start mysqlbinlog with --force if you want to attempt "
+                "Start mysqlbinlog with --force-read if you want to attempt "
                 "to read encrypted binlog without decryption.";
```

Comment:
Not related to the issue itself, but just spotted this:
```suggestion
                "to read an encrypted binlog without decryption.";
```

Reply:
Addressed this change